clippy.toml

```diff
@@ -2,3 +2,4 @@ disallowed-methods = []
 disallowed-types = []
 enforced-import-renames = []
 large-error-threshold = 256
+enum-variant-size-threshold = 256
```

Comment:
FYI: for some reason Clippy started to complain about "large size difference between variants" for multiple enums, even though the default value for `enum-variant-size-threshold` doesn't seem to have changed between the compiler versions. I decided to set it to 256 explicitly, for consistency with `large-error-threshold`, this silenced the complaints.